classes/publication/PublicationCategory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace PKP\publication;
+
+use Illuminate\Database\Eloquent\Model;
+
+class PublicationCategory extends Model
+{
```

Comment:
Use the [`HasCamelCasing`](https://github.com/pkp/pkp-lib/blob/main/classes/stageAssignment/StageAssignment.php#L34) trait. ([Documentation](https://github.com/kirkbushell/eloquence?tab=readme-ov-file#camel-case-all-the-things))

Reply:
@asmecher I noticed that the HasCamelCasing trait isn’t present in stable-3_4_0 (only in main). Could you please clarify it?